CONTRIBUTING.md

```diff
@@ -123,6 +119,54 @@ use a shared working branch strategy.
 5. Resolve any merge conflicts.
 6. When your work is complete, merge the shared working branch into `develop` via a PR.
 
+### Working with docs-website-private
```

Comment:
All of the workflow info seems more appropriate for the docs-website-private contributing doc. I could see pointing to the docs-website-private repo here, with a simple paragraph that says:
 
For New Relic employees working on pre-release content that needs to remain private, we have a private docs repo (link). Before you begin creating content in the private repo, see the contributing guidelines (link); keeping `docs-website` and `docs-website-private` in sync requires a few extra steps.

Reply:
updated